encrypted-media-respec.html

```diff
@@ -2111,20 +2099,44 @@
                     Destroy these license(s) and/or key(s).
                   </p>
                 </li>
-                <li>
-                  <p>
-                    If this object's <var>session type</var> is <a def-id="persistent-usage-record-session"></a>, store the <var>record of key usage</var>.
-                  </p>
-                </li>
               </ol>
             </li>
-            <li><p>Run the <a def-id="update-key-statuses-algorithm"></a> algorithm on the <var>session</var>, providing an empty sequence.</p></li>
-            <li><p>Run the <a def-id="update-expiration-algorithm"></a> algorithm on the <var>session</var>, providing <code>NaN</code>.</p></li>
-            <li><p>Let <var>promise</var> be the <a def-id="closed"></a> attribute of the <var>session</var>.</p></li>
-            <li><p>Resolve <var>promise</var>.</p></li>
+            <li><p>Run the <a def-id="session-closed-algorithm"></a> algorithm on the <var>session</var>.</p></li>
           </ol>
         </section>
       </section>
+      
+      <section id="session-closed">
```

Comment:
Do you intend for this algorithm to be at a higher level than all the other algorithms? I think line 2107 needs to be moved below it.